src/redisearch_rs/build_utils/src/lib.rs

```diff
@@ -156,6 +156,7 @@ pub fn generate_c_bindings(
     let includes = [
         root.join("deps").join("RedisModulesSDK"),
         root.join("src"),
+        root.join("src").join("inverted_index"), // Add missing inverted_index include
```

Comment:
This adds `inverted_index` to everything which uses this utils crate. We won't want that, but will only want the iterators bencher to have this include.